tests/tracking/test_mlflow_artifacts.py

```diff
@@ -245,3 +246,49 @@ def test_mlflow_artifacts_example(tmpdir):
         check=True,
         cwd=os.path.join(root, "examples", "mlflow_artifacts"),
     )
+
+
+def get_artifacts_from_rest_api(url, run_id, path=None):
+    if path:
+        resp = requests.get(url, params={"run_id": run_id, "path": path})
+    else:
+        resp = requests.get(url, params={"run_id": run_id})
+    assert resp.status_code == 200
+    return json.loads(resp.content.decode("utf-8"))
```

Comment:
```suggestion
    resp.raise_for_status()
    return resp.json()
```

nit: can we use `.raise_for_status` and `.json`?

Reply:
definitely!